### PR TITLE
add connect() from utils to avoid ssh-keygen for hmc and vios

### PR DESCRIPTION
**Pull request by smruti77**
to avoid password of hmc and vios during test run chnage for
dlpar_vscsi.py,veth_dlpar.py,veth_dlpar.py,network_virtualization.py

Signed-off-by: bismurti bidhibrata pattajoshi <bbidhibr@in.ibm.com>

### CODE REVIEW

**Comment by smruti77** [comment]
JOB ID     : c7cf78607a51a293ebbae80feae3b09a1cb62cd0
JOB LOG    : /root/avocado/job-results/job-2020-07-02T01.33-c7cf786/job.log
 (1/3) dlpar.py:DlparPci.test_dlpar;lpar_2-url-015c: PASS (52.80 s)
 (2/3) dlpar.py:DlparPci.test_drmgr_pci;lpar_2-url-015c: FAIL: drmgr operation r fails for PCI (8.95 s)
 (3/3) dlpar.py:DlparPci.test_drmgr_phb;lpar_2-url-015c: PASS (31.12 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2020-07-02T01.33-c7cf786/results.html
JOB TIME   : 93.00 s


JOB ID     : ac63d432dfc50eb72dd6f9e382515b435d236288
JOB LOG    : /home/ioci/avocado-fvt-wrapper/results/job-2020-07-02T00.54-ac63d43/job.log
 (1/1) dlpar_vscsi.py:DlparTest.test_dlpar;run-d054: PASS (122.23 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/ioci/avocado-fvt-wrapper/results/job-2020-07-02T00.54-ac63d43/results.html
JOB TIME   : 130.71 s

JOB ID     : 3b7440a3dfa62afa19f2ebc920aa0a6d37efac7a
JOB LOG    : /home/ioci/avocado-fvt-wrapper/results/job-2020-07-02T02.39-3b7440a/job.log
 (1/1) veth_dlpar.py:VethdlparTest.test_dlpar;run-51c2: PASS (52.47 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/ioci/avocado-fvt-wrapper/results/job-2020-07-02T02.39-3b7440a/results.html
JOB TIME   : 52.55 s

File: io/disk/dlpar_vscsi.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ def setUp(self):
         self.vios_pwd = self.params.get('vios_pwd', '*', default=None)
         self.session = Session(self.vios_ip, user=self.vios_user,
                                password=self.vios_pwd)
+        self.session.connect()
```
Comment (abdhaleegit) [inline io/disk/dlpar_vscsi.py:42]: should this also be 
```
if not self.session.connect():
         self.cancel("failed connecting to vios")
```

Comment (smruti77) [reply io/disk/dlpar_vscsi.py:42]: @abdhaleegit its always failing for vios , if conneting also so i avoid to use if.

Comment (abdhaleegit) [reply io/disk/dlpar_vscsi.py:42]: if it always fails than why to add ? for vios

**Comment by abdhaleegit** [comment]
LGTM, you can merge

**Pull request merged**